pdk/1.0/modules/ROOT/pages/policies-pdk-configure-features-http-request.adoc

```diff
@@ -55,7 +64,7 @@ properties:
 
 To learn more about defining parameters, see xref:policies-pdk-create-schema-definition.adoc[]
 
-In Rust, you can access the defined external service in the `Config` struct. To perform requests, call the HTTP client in your policy similar to the following example:
+You must define your service as a parameter in you schema definition to make HTTP requests. To perform requests, call the HTTP client in your policy similar to the following example:
```

Comment:
Why do we repeat this phrase again here? Here we would want to mention the `Config` struct

Reply:
Sorry about this one, I started editing a different section in last round.